tests/experimental/datasources/test_config.py

```diff
@@ -347,7 +348,7 @@ def test_yaml_config_round_trip(
 
 
 def test_yaml_file_config_round_trip(
-    inject_engine_lookup_double, tmp_path: pathlib.Path, from_yaml_gx_config: GxConfig
+    inject_engine_lookup_double, tmp_path: PathStr, from_yaml_gx_config: GxConfig
```

Comment:
```suggestion
    inject_engine_lookup_double, tmp_path: pathlib.Path, from_yaml_gx_config: GxConfig
```
Please revert this. `pathlib.Path` is the correct type. `PathStr` is `str | pathlib.Path` union.

Reply:
@Kilo59 I will look to see where we need the alias and where we do not.  Thanks.